src/components/Table/index.js

```diff
@@ -202,7 +202,11 @@ function Table({ columns, data, isCheckable, onCompare, loadingData }) {
               {page.map(row => {
                 prepareRow(row);
                 return (
-                  <tr className="pf-m-hoverable" {...row.getRowProps()}>
+                  <tr
+                    className="pf-m-hoverable"
+                    {...row.getRowProps()}
+                    onClick={onRowClick ? () => onRowClick(row.original) : null}
```

Comment:
What if we did  `onClick={onRowClick && () => onRowClick(row.original)}` for a single condition?